xdist/dsession.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, config):
         self._failed_collection_errors = {}
         self._active_nodes = set()
         self._failed_nodes_count = 0
-        self._max_slave_restart = self.config.getoption('max_slave_restart')
+        self._max_slave_restart = self.config.getoption('max_worker_restart')
```

Comment:
It would be better if the code was also changed to use worker rather than slave. Does this code also support the old command line option to stop everything breaking during transition?

Reply:
No, I just renamed to new method. Do you need support old value? I'll add support

Reply:
I'm not in charge here, but I think it would be best if a deprecation notice is used for the old command so that it is still supported for a couple of releases before being dropped.

Reply:
> It would be better if the code was also changed to use worker rather than slave

I didn't change another code bacause it must be changed in pytest repo else. Do you think I need rename all code?

Reply:
I meant, can we stop using "slave" in the xdist code everywhere, and not just in the messages seen by users? Using worker externally and slave internally seems like a weird inconsistency when the aim is to remove slave nomenclature everywhere. I wasn't saying anything about other pytest repos.

PS Thanks for tackling this. I really do appreciate this getting done since I haven't had a chance to have a go myself.

Reply:
Ok, I'll remove all "slave" from code, but there is one file which uses in pytest. So, I need to create commit in pytest repo else

Reply:
@feuillemorte you mean the `slaveinput` attribute of the `config` object? I think for that case we can have both names, `slaveinput` and `workerinput` which point to the same dictionary so we can keep backward compatibility.

Reply:
@nicoddemus ok, thank you!